src/pinns_pde_solve.jl

```diff
@@ -782,9 +783,9 @@ function DiffEqBase.discretize(pde_system::PDESystem, discretization::PhysicsInf
     end
 
     function loss_function_(θ,p)
```

Comment:
not quite. We want the `p` here of the parameters to be appended to `theta` so that they are optimized simultaneously, not as hyperparameters.